plugins/GSdx/GSSettingsDlg.h

```diff
@@ -64,6 +64,20 @@ class GSHacksDlg : public GSDialog
 	}
 };
 
+class GSOSDDlg : public GSDialog
+{
+	int transparency;
```

Comment:
Same as above. I copied how it worked in the hacks/shader dialogs. It is used in all the GSOSDDlg functions to keep track of the setting, so it is convenient.